### PR TITLE
Add helpful PowerShell example Get-AppAUMID

### PR DESCRIPTION
**Pull request by djust270**
## Description
Added the PowerShell example Get-AppAUMID. This is very helpful when customizing the start menu layout and needing to add non UWP apps. 

## Why
To help other ITPros by enriching the documentation with real world examples that I use in my day to day work. 

## Changes

This PR adds the PowerShell function "Get-AppAUMID" and examples for using it.

### CODE REVIEW

File: windows/configuration/find-the-application-user-model-id-of-an-installed-app.md [hunk 1]
```diff
@@ -109,3 +109,40 @@ listAumids("CustomerAccount")
 # Get a list of AUMIDs for all accounts on the device:
 listAumids("allusers")
 ```
+
+## Example
+The following code sample creates a function in Windows PowerShell that returns the AUMID of any application currently listed in the Start Menu
```
Comment (JohanFreelancer9) [inline windows/configuration/find-the-application-user-model-id-of-an-installed-app.md:114]:
```suggestion

The following code sample creates a function in Windows PowerShell that returns the AUMID of any application currently listed in the Start menu.
```

File: windows/configuration/find-the-application-user-model-id-of-an-installed-app.md [hunk 2]
```diff
@@ -109,3 +109,40 @@ listAumids("CustomerAccount")
...
+The following Windows PowerShell commands demonstrate how you can call the Get-AppAUMID function after you've created it.
+
+```powershell
+# Get the AUMID for OneDrive
+Get-AppAUMID -AppName OneDrive
+
+# Get the AUMID for Microsoft Word
+Get-AppAUMID -AppName Word
+
+# List all apps and their AUMID in the Start Menu
```
Comment (JohanFreelancer9) [inline windows/configuration/find-the-application-user-model-id-of-an-installed-app.md:145]:
```suggestion
# List all apps and their AUMID in the Start menu
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @djust270, kindly commit the suggested changes. Many thanks.

**Comment by djust270** [comment]
I committed the suggested changes @JohanFreelancer9. Thank you.

**Pull request merged**